6-pubsub/test/_test-config.worker.js

```diff
@@ -14,10 +14,10 @@
 'use strict';
 
 const path = require(`path`);
-const PROJECT_ID = process.env.GCLOUD_PROJECT;
 const TESTNAME = `6-pubsub`;
 const PORT = 8091;
 const VERSION = `${process.env.GAE_VERSION || TESTNAME}`;
+const PROJECT_ID = process.env.GCLOUD_PROJECT;
```

Comment:
Why keep it here?

Reply:
Because we have these e2e tests: 

` module.exports.testUrl = `https://${VERSION}-dot-worker-dot-${PROJECT_ID}.appspot.com`;
}`